src/ReshowcaseUi.re

```diff
@@ -939,7 +900,18 @@ module DemoUnitFrame = {
           ~border="none",
           (),
         )}
-      />
+        onLoad={event => {
+          let iframe = event->React.Event.Synthetic.target;
+          let body =
+            iframe##contentWindow##document->Option.flatMap(d => d##body);
+          setBody(_ => body);
+          onLoad(iframe##contentWindow);
+        }}>
+        {switch (body) {
+         | None => React.null
+         | Some(body) => ReactDOM.createPortal(children, body)
+         }}
+      </iframe>
```

Comment:
This is the core change. Instead of referencing iframe with demo, we just render demo using portal in persistent iframe

Reply:
Have you thought about how this change might affect stateful changes that the iframe keeps from one demo to another? E.g. if some iframe loads some styles in the page `head` then they might "leak" from one demo to the next.